test/uxCapture.test.js

```diff
@@ -91,6 +85,10 @@ describe('UXCapture', () => {
 			ExpectedMark.clearExpectedMarks();
 			onMark.mockClear();
 			onMeasure.mockClear();
+
+			UXCapture.create({ onMark, onMeasure });
```

Comment:
These tests are not set up correctly. You are already calling `UXCapture.create()` and `UXCapture._clearMarksAndMeasures()` in the `beforeEach()` binding  (https://github.com/meetup/ux-capture/pull/16/files#diff-cceed845ff270e266b3b8b9057751fbeR37). `beforeEach` executes before each individual unit test. You're then calling them a second time in each of these test cases. Ideally, if you can have them live only in the `beforeEach()` binding that'd be less code and easier to read.

Reply:
Got it. Looks like I should've removed more `UXCapture.create()` calls when I moved them, `from beforeAll()` to `beforeEach()`

Reply:
Actually, my bad, this particular set of tests is testing various ways `UXCapture.create()` is called so I removed generic calls from beforeEach() instead - just copied that block to all tests without giving it enough though before.